src/scikit_package/cli/create.py

```diff
@@ -20,3 +83,30 @@ def package(args):
         cookie.run(f"{SKPKG_GITHUB_URL}-conda-forge")
     elif subcmd == "manuscript":
         cookie.run(f"{SKPKG_GITHUB_URL}-manuscript")
+    items_after_run = [f for f in Path().cwd().iterdir()]
+    if args.update:
+        print("Create the package in the update mode.")
```

Comment:
how about `f"Updating the existing package {package_name}."`   Why not do all the validations all at once and then decide which message to send them.  I.e., send a different message if it is going to fail.